nsot/conf/settings.py

```diff
@@ -52,6 +53,7 @@
 AUTHENTICATION_BACKENDS = (
     'nsot.middleware.auth.EmailHeaderBackend',
     'django.contrib.auth.backends.ModelBackend',
+    'guardian.backends.ObjectPermissionBackend',
```

Comment:
We'll need `nsot.middleware.auth.NsotObjectPermissionsBackend` defined here instead.

Guardian will raise a warning. It can be silenced by adding this to settings:

```
 SILENCED_SYSTEM_CHECKS = ['guardian.W001']
```

Reply:
Done